My changes

DRLoureiro opened the pull request:
pull 3

   made by me:
mudkip - done
marshtomp - done
swampert - done
taillow - done
swellow - done
carvanha - done
sharpedo - done
torkoal - done
mawile - done
meditite - done
medicam - done
shinx - done
luxio - done
luxray - done
druddigon - done
bouffalant - done
stuffel - done
beweare - done
chewtle - done
drednaw - done
clobbopus - done
grapploct - done

   made by alex:
frillish - done
jellicent - done
pinsir - done
carnivine - done
fuecoco - done
crocalor - done
skeledirge - done
drifloon - done
drifblimb - done
cufant - done
copperajah - done
wimpod - done
golisopod - done
electrike - done
manectric - done
passimian - done
rowlet - done
dartrix - done
decidueye - done
h-decidueye - done

DRLoureiro committed:
commit message

DRLoureiro committed:
commit message

DRLoureiro committed:
commit message

MichaelFacci reviewed (changes requested):
All of Alex's submissions have severe syntax errors. Please make sure there are no spaces, and that everything follows "MOVE_NAME_ALL_CAPS" format instead of "MOVE_Name Weird Syntax"

DRLoureiro committed:
Update levelupdata.s

strenght -> strength
upper case where it was supposed to
_ where it was supposed to

MichaelFacci committed:
Merge branch 'iridium-dev' into my-changes
